### PR TITLE
Tidied up the test class and added the spec for the API.

### PR DESCRIPTION
**Pull request by cjllanwarne**
The tests were broken/terrible before. You can pretty much ignore the LHS of the diff and just comment as though they were new files.

NB: In case you're tempted to cross-reference against the code... no, these tests don't all pass yet.

### CODE REVIEW

File: src/main/scala/thurloe/service/ApiDataModels.scala [hunk 1]
```diff
@@ -1,6 +1,8 @@
 package thurloe.service
 
+import akka.serialization.Serialization
 import com.wordnik.swagger.annotations.{ApiModel, ApiModelProperty}
+import spray.http.HttpEntity
 import spray.json.DefaultJsonProtocol
 
 import scala.annotation.meta.field
```
Comment (mcovarr) [inline src/main/scala/thurloe/service/ApiDataModels.scala]: are these new imports being used?

File: src/main/scala/thurloe/service/ThurloeService.scala [hunk 2]
```diff
@@ -70,27 +73,13 @@ trait MyService extends HttpService {
...
-  }
-
-  val myRoute = path("") {
-    get {
-      respondWithMediaType(`text/html`) { // XML is marshalled to `text/xml` by default, so we simply override here
+      respondWithMediaType(`text/plain`) {
         complete {
-          <html>
-            <body>
-              <h1>Say hello to <i>Thurloeeeeeeee</i>!</h1>
```
Comment (mcovarr) [inline src/main/scala/thurloe/service/ThurloeService.scala]: This was my favorite endpoint.  :cry:

Comment (geoffjentry) [inline src/main/scala/thurloe/service/ThurloeService.scala]: I'm writing a ticket to re-add it!

**Comment by mcovarr** [comment]
:+1:

**Pull request merged**